src/medicine-selector/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Topbar from '../components/topbar';
+import ContainerPastis from "../components/contenedor-medicamentos";
+import Button from '../components/button';
+import Searcher from '../components/searcher';
+import styles from "./style.module.scss";
+
+
+
+class MedicineSelector extends React.Component {
+  render() {
+    return(
+      <div>    
+
+        <Topbar/>
+         <div className={styles.contenedorPrincipal}>
+          <Searcher contBuscador="¿Que medicamentos estas buscando?" title="MEDICAMENTOS UTILIZADOS"/>
+          <ContainerPastis/>
+          <Button title="Continuar"/>
+        </div>
+
+    </div>

```

Comment:
Procura no poner espacios de más.
```suggestion
      <div>
        <Topbar/>
         <div className={styles.contenedorPrincipal}>
          <Searcher contBuscador="¿Que medicamentos estas buscando?" title="MEDICAMENTOS UTILIZADOS"/>
          <ContainerPastis/>
          <Button title="Continuar"/>
        </div>
    </div>

```